### ISSUE TITLE
Union panic error (epff)

### ISSUE THREAD

**Issue by saviodsouza** [rclone/rclone#5000]
#### What is the problem you are having with rclone?

after sync the panic error messsage.

i have another union in the config for another union like

```
create_policy = rand
action_policy = rand
search_policy = all
```

and this works well

#### What is your rclone version (output from `rclone version`)

rclone v1.54.0
- os/arch: linux/amd64
- go version: go1.15.7

#### Which OS you are using and how many bits (e.g. Windows 7, 64 bit)

Ubuntu 18.04 Server 64bit

####  Which cloud storage system are you using? (e.g. Google Drive)

source: Nas mount
remote : 2 one drive for business accounts under union

#### The command you were trying to run (e.g. `rclone copy /tmp remote:tmp`)

```
rclone sync -Pv /mnt/nas/ UN-combo:
```

```
UN-combo:
type = union
upstreams = OD-one:backup OD-two:backup
action_policy = epff
search_policy = ff
create_policy = mfs
```

#### A log from the command with the `-vv` flag (e.g. output from `rclone -vv copy /tmp remote:tmp`)

```
......... trimmed......
Copied (new)
panic: close of closed channel

goroutine 5083 [running]:
github.com/rclone/rclone/backend/union/policy.(*EpFF).epff.func2(0xc000fbd380, 0x1)
	github.com/rclone/rclone/backend/union/policy/epff.go:42 +0x90
created by github.com/rclone/rclone/backend/union/policy.(*EpFF).epff
	github.com/rclone/rclone/backend/union/policy/epff.go:37 +0x1a5
```

**Comment by ncw** [comment]
Can you give this a go to see if it fixes it?

[v1.55.0-beta.5164.ba7b5bd87.fix-5000-union-epff](https://beta.rclone.org/branch/fix-5000-union-epff/v1.55.0-beta.5164.ba7b5bd87.fix-5000-union-epff/) on branch [fix-5000-union-epff](https://github.com/rclone/rclone/tree/fix-5000-union-epff) (uploaded in 15-30 mins)

Thanks

**Comment by saviodsouza** [comment]
Give me another day to confirm...

Looks ok for now...

Thank you so much

**Comment by saviodsouza** [comment]
This patch is working...

This can be merged.

**Comment by ncw** [comment]
I've merged this to master now which means it will be in [the latest beta](https://beta.rclone.org) in 15-30 mins and released in v1.55
